kernel/common/ezautolinkoperator.php

```diff
@@ -42,17 +42,48 @@ function formatUri( $url, $max )
         }
         return "<a href=\"$url\" title=\"$url\">$text</a>";
     }
+    
+    static function formatUriSimple( $url, $max, $method = '' )
```

Comment:
Do we really need a separate function? We really don't think so. We also do not care for the current name as vague. We think you features would be better placed within the original function instead.